gclient/src/api/calls.rs

```diff
@@ -204,6 +204,72 @@ impl GearApi {
             .await
     }
 
+    /// Migrates an active program identified by `src_program_id` onto another
+    /// node identified by `dest_node_api` and returns the migrated program
+    /// identifier.
+    pub async fn migrate_program(
+        &self,
+        src_program_id: ProgramId,
+        dest_node_api: &GearApi,
+    ) -> Result<ProgramId> {
+        // TODO: Ideally we have to check program is not already there (via gprog, for
+        // example). Do we want to override it? If yes, then we have to check
+        // migration is done to another node (network). How?
+        // Another solution might be that we crate a brand new ProgramId which would
+        // allow us to migrate program even withing the same node/network
```

Comment:
Have discussed, need to add an issue and reference to it here if the check will not be added to this PR.

Reply:
@shamilsan , @breathx I have fixed it in this PR. Please check this commit https://github.com/gear-tech/gear/pull/2331/commits/02f9036677434f3f733dbeef5ee1ff3e1fb6d49d. Found and fixed one bug while was doing this.